src/content/docs/new-relic-solutions/new-relic-one/ui-data/collaborate/collaborate-with-teammates.mdx

```diff
@@ -0,0 +1,97 @@
+
+<Callout variant="important">
+The functionality described below is currently in preview. If you don't yet have access but would like to test the commenting and discussion functionality [please sign up here](https://one.newrelic.com/discussions).
+</Callout>
+
+We're making it easier to collaborate with your teammates by building discussion and commenting tools into New Relic. You can now add comments on any page. Our integrations with Slack, MS Teams (coming soon), and email let you share links and screenshots, and communicate with your teammates directly from the New Relic UI without switching applications. Our collaboration tools allow you to identify problems and find solutions in one place. Moving the discussion closer to your data streamlines your workflows, improves efficiency, and reduces MTTR.
+
+## How it works [#collaborate]
+
+<Video
+ type="wistia"
+ id="m4b6y854wi"
+/>
+
+We've integrated discussion tools directly into New Relic so you can: 
+
+* **Start discussions in context at the source.** When you comment on a dashboard or an error in New Relic, that comment stays with that dashboard or error.
+* **Sync discussions with Slack.**  With our Slack integration you can share links, screenshots, and comments to Slack in seconds. All teammates  can reply from Slack and discussions stay in sync, no matter where teammates contribute. [Learn more about the Slack integration](https://docs-preview.newrelic.com/docs/collaboration-slack-integration).
+* **Create shared knowledge and history.** Collaborating within the New Relic platform will allow your team to build a searchable body of shared knowledge.
+## Comment on any page [#comment-on-any-page]
+
+On the global navigation, there's a new “add comment” icon on most pages. A green dot on the icon means there is actually an open discussion on the page. A gray dot means there are closed discussions that can perhaps answer new questions or help resolve current issues. 
+
+<img
+  title="Comment on any page "
+  alt="A screenshot of the UI that depicts commenting on a specific page."
+  src={green_dot}
+/>
+
+If you want to ask a question, start a discussion, or share a permalink or screenshot to Slack, click the icon to open the comment panel. Type your comment and start a discussion on the page you're looking at. When you comment on a dashboard or an error in New Relic, that comment stays with your data. You don't need to spend time repeating information that can be seen at a glance, and everybody knows where to find an active, solution-oriented conversation.
+
+You may notice the “add comment” icon is disabled on some pages. By design, in a multi-account organization you can't discuss pages that either don't have data associated with a specific account. You also won't be able to discuss on pages that present data belonging to multiple accounts in a single view. Simply selecting a specific account or clicking through to a specific item or entity will usually show the “add comment” icon.
```

Comment:
The "present" can be read incorrectly (present being indicative of time instead of indicative of an action), what do you think about changing it to just "show" or something like that?

Reply:
I think you can cut Simply from the last sentence

Reply:
thanks :)